src/main/java/competition/subsystems/pose/PoseSubsystem.java

```diff
@@ -75,6 +78,11 @@ protected void updateOdometry() {
     }
 
     private SwerveModulePosition[] getSwerveModulePositions() {
+        // if we have simulated data, return that directly instead of asking the
+        // modules
+        if (simulatedModulePositions.isPresent()) {
```

Comment:
Do you have any better ideas for how to handle this interface? Felt icky hacking it into the code like this but was expedient.